internal/commands/verify.go

```diff
@@ -71,12 +72,25 @@ func NewVerifyCommand(ctx context.Context) *cobra.Command {
 	return &cmd
 }
 
-func runVerification(ctx context.Context, path string, trace bool) ([]CheckResult, error) {
+func runVerification(ctx context.Context, path string, logger *log.Logger, trace bool) ([]CheckResult, error) {
+	logger.Printf("Attempting path \"%s\"", path)
+
 	regoFiles, err := policy.ReadFilesWithTests(path)
 	if err != nil {
 		return nil, fmt.Errorf("read rego test files: %s", err)
 	}
 
+	if len(regoFiles) == 0 {
+		logger.Printf("No policies found in dir \"%s\"", path)
```

Comment:
Most commands now returned formatted errors instead of logging directly in the command, it would be better if we stuck to that convention.

Reply:
I think this boils down to the question: Do we want to consider conftest not finding any policies as an error code of 0 or non-zero.

1. Return an error with this message, saying no policies were found. Error code would be non-zero and users/CI pipelines would most likely fail.

2. Have conftest return a zero error code, indicating success, but let the user know it didn't verify any policies.